lib/discovery.ts

```diff
@@ -59,13 +58,14 @@ function makeMethod (schema, method, context) {
       params: params,
       requiredParams: method.parameterOrder || [],
       pathParams: getPathParams(method.parameters),
-      context: context
+      context: context,
+      mediaUrl: null
```

Comment:
It was done just for TypeScript, but this object literal is what formed the implicit type the compiler was checking against.  When I do a proper typing PR, I will move this to an interface with an optional parameter.